doc-content/enterprise-only/project-data/reviewing-change-requests-proc.adoc

```diff
@@ -0,0 +1,24 @@
+[id='reviewing-change-requests-proc_{context}']
+
+= Working with change requests
+You can view change requests for any branch that you have access to. You must have administrator permissions to accept a change request.
+
+.Prerequisites
+* You have more that one branch of a {CENTRAL} project.
+
+.Procedure
+. In {CENTRAL}, go to *Menu* -> *Design* -> *Projects* and select a space and project.
+. On the project page, verify that you are on the correct branch.
++
+image::getting-started/change-branch.png[Selecting a branch menu]
```

Comment:
This image in the draft does not seem updated since I cannot see the _Change Requests_ tab.

Reply:
Yes, well spotted :-) Because screen shots are so time consuming my plan was to write the text first for your review then update the screen shots.

Reply:
I have now updated the images.